includes/controllers/ApiController.php

```diff
@@ -117,4 +121,32 @@ public function getAllPages()
         }
         return new ApiResponse(empty($pagesWithTag) ? null : $pagesWithTag);
     }
+
+    /**
+     * @Route("/api/pages/{id}",options={"acl":{"public"}})
```

Comment:
Je me pose la question de savoir si c'est intuitif de filtrer par id de base sql. J'étais plus partie sur
```suggestion
     * @Route("/api/pages/{tag}",options={"acl":{"public"}})
```
Qu'en penses-tu ? @mrflos 

Finalement, je me demande si pour ce cas, et pour rester dans l'esprit des autres routes api, il ne faudrait pas une autre route dont il faudrait trouver un nom (qui pourrait être du genre `api/pagesById/{id}` pour éviter les confusions possibles

Reply:
Je pense qu'il ne faudrait qu'une route d’accès aux pages, mais en effet, c'est plus commun et lisible de chercher par tag que par id ...
peut etre passer l'id (ou le datetime de revision) en option GET ? `/api/page/<MaPage>&id=<id>` ou `/api/page/<MaPage>&date=<date de la modif>`

Reply:
oui je comprend. En REST, la base c'est de filtrer par ID, mais c'est vrai que dans le contexte de YesWiki filter une page par ID ne semble pas le truc par default. ok pour changer si besoin

Reply:
ok parfait je m'en occupe